dask/dataframe/tests/test_multi.py

```diff
@@ -2129,10 +2130,26 @@ def test_concat5():
     "known, cat_index, divisions",
     [
         (True, True, False),
-        (True, False, True),
+        pytest.param(
+            True,
+            False,
+            True,
+            marks=pytest.mark.skipif(
+                PANDAS_GE_220,
+                reason="fails on pandas dev: https://github.com/dask/dask/issues/10558",
+            ),
+        ),
         (True, False, False),
         (False, True, False),
-        (False, False, True),
+        pytest.param(
+            False,
+            False,
+            True,
+            marks=pytest.mark.skipif(
```

Comment:
can we do xfail with strict? Just want to see if this changes upstream at some point